web/src/components/DandisetsPage.vue

```diff
@@ -14,17 +14,25 @@
         Sort By:
       </div>
       <v-chip-group
-        v-model="selection"
+        :value="Number(sortOption)"
```

Comment:
```suggestion
        :value="sortOption"
```
Seems redundant to me

Reply:
You're right, left over from previous work.